src/theory/inference_id.h

```diff
@@ -66,6 +66,13 @@ enum class InferenceId
   ARITH_BB_LEMMA,
   ARITH_DIO_CUT,
   ARITH_DIO_DECOMPOSITION,
+  // unate lemma during presolve
+  ARITH_UNATE,
+  // row implication
+  ARITH_ROW_IMPL,
```

Comment:
Can you elaborate what this means?

Reply:
I actually have no idea.  @alex-ozdemir may know since he did proofs for this part?